Brokkr/src/main/cpp/commands/MoveArmIntake.cpp

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+#include "commands/MoveArmIntake.h"
+#include "commands/IntakeSpin.h"
+#include "commands/ArmPosition.h"
+
+// NOTE:  Consider using this command inline, rather than writing a subclass.
+// For more information, see:
+// https://docs.wpilib.org/en/stable/docs/software/commandbased/convenience-features.html
+MoveArmIntake::MoveArmIntake(Arm& arm, Claw& claw, double armAngle) 
+: CommandHelper(IntakeSpin(mClaw))
+,mArm(arm)
+,mClaw(claw)
+,mArmAngle(armAngle)
```

Comment:
Remove all of the member variables. This command does not need to hold any of these variables